docs/ref/models/querysets.txt

```diff
@@ -2246,6 +2251,7 @@ them, but it has a few caveats:
   extra query per ancestor.
 * When an individual batch contains duplicates, only the first instance in that
   batch will result in an update.
+* The return value of updated objects is expected to be same as the number of objects passed. Although, it may be lesser in case there are duplicate objects passed in the same batch.
```

Comment:
```suggestion
* The return value of updated objects is expected to be same as the number of 
  objects passed. Although, it may be lesser in case there are duplicate objects 
  passed in the same batch.
```